app/models/second_review_turnaround.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
+#
+# Table name: second_review_turnarounds
+#
+#  id                :bigint           not null, primary key
+#  value             :integer
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#  review_request_id :bigint           not null
+#
+# Indexes
+#
+#  index_second_review_turnarounds_on_review_request_id  (review_request_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (review_request_id => review_requests.id)
+#
+
+class SecondReviewTurnaround < ApplicationRecord
```

Comment:
@SandroDamilano @santiagovidal @santib  wdyt about the name of the table/model? i couldn't imagine any better 😅

Reply:
It doesn't sound bad. It represents the review turnaround of every second review after all, doesn't it?

Reply:
I'm ok with it, but maybe the actual entity is not just the second review, but the "last awaited review"? Like, the last of the reviews that were required to be able to merge it (best case scenario when there are no comments). I'm thinking that if tomorrow there are projects that require different amounts of reviews this modeling won't suffice the requirements.

That said, I'm ok leaving it like this. Chances are that we won't need to do anything like that 😄